### ISSUE TITLE
Doesn't work for me

### ISSUE THREAD

**Issue by redreceipt** [teaxyz/cli#183]
When I try to install tea and then run the rust interpreter, it fails:

```sh
Projects % sh <(curl tea.xyz)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
  the latest version of tea was already installed
  
  │ /Users/michael/.tea/tea.xyz/v0.13.5/bin/tea

Projects % tea +rust-lang.org
tea: gnome.org/libxml2@2.10.3 downloading 763 KiB
error: http (spilt-tea-404)
No such file or directory (os error 2), open '/Users/michael/.tea/tea.xyz/var/www/gnome.org∕libxml2-2.10.3+darwin+aarch64.tar.xz'
```

**Comment by mxcl** [comment]
So I don’t get this exactly, but it seems like: try again… might work?

@jhheider

**Comment by redreceipt** [comment]
maybe this will give you a little more info? `www` isn't there:

```sh
~ % sh <(curl tea.xyz) +nodejs.org   
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9259  100  9259    0     0  31083      0 --:--:-- --:--:-- --:--:-- 31493

tea: unicode.org@71.1.0 downloading 18.1 MiB
error: http (spilt-tea-404)
No such file or directory (os error 2), open '/var/folders/cl/kg5548xd23sght605mqpnb500000gn/T/tea-XXXXXX.fwpKceD3/tea.xyz/var/www/unicode.org-71.1.0+darwin+aarch64.tar.xz'

  
  │ powered by tea https://tea.xyz; brew2 for equitable open-source

~ % cd /var/folders/cl/kg5548xd23sght605mqpnb500000gn/T/tea-XXXXXX.fwpKceD3/tea.xyz/var/www
cd: no such file or directory: /var/folders/cl/kg5548xd23sght605mqpnb500000gn/T/tea-XXXXXX.fwpKceD3/tea.xyz/var/www
~ % cd /var/folders/cl/kg5548xd23sght605mqpnb500000gn/T/tea-XXXXXX.fwpKceD3/tea.xyz/var  
var % ls
pantries pantry
```

**Comment by AkiraHasegawa1997** [comment]
Hi, I met same problem on MacBook Pro M1. I'll paste the verbose log below.
```terminal
➜ ls ~/.tea/tea.xyz/    
tmp     v*      v0      v0.13   v0.13.5 var
 
➜ ls ~/.tea/tea.xyz/var
pantries pantry

➜ tea -vvv +rust-lang.org 
{
  args: { args: [], pkgs: [ { project: "rust-lang.org", constraint: * } ] },
  flags: {
    verbosity: 3,
    magic: true,
    json: false,
    numpty: false,
    sync: false,
    verbose: true,
    debug: true,
    muggle: false,
    silent: false
  }
}
/Users/hasegawa/.tea/tea.xyz/v0.13.5/bin/tea
tea: resolving package graph
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/rust-lang.org/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/rust-lang.org/cargo/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/rust-lang.org/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/tea.xyz/gx/cc/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/zlib.net/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/rust-lang.org/cargo/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/git-scm.org/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/gnu.org/gettext/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/gnome.org/libxml2/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/curl.se/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/openssl.org/package.yml"
}
{
  readYAML: "/Users/hasegawa/.tea/tea.xyz/var/pantry/projects/perl.org/package.yml"
}
{ installations: [], versions: [], version: undefined }
{ project: "gnome.org/libxml2", versions: [ 2.10.3 ] }
{ installations: [], versions: [], version: undefined }
{ project: "openssl.org", versions: [ 1.1.118 ] }
{ installations: [], versions: [], version: undefined }
{ project: "gnu.org/gettext", versions: [ 0.21.1 ] }
{ installations: [], versions: [], version: undefined }
{ project: "curl.se", versions: [ 7.85.0, 7.86.0 ] }
{ installations: [], versions: [], version: undefined }
{ project: "perl.org", versions: [ 5.36.0 ] }
{ installations: [], versions: [], version: undefined }
{ project: "tea.xyz/gx/cc", versions: [ 0.1.0, 0.1.1 ] }
{ installations: [], versions: [], version: undefined }
{ project: "zlib.net", versions: [ 1.2.13 ] }
{ installations: [], versions: [], version: undefined }
{ project: "git-scm.org", versions: [ 2.38.1 ] }
{ installations: [], versions: [], version: undefined }
{ project: "rust-lang.org", versions: [ 1.64.0, 1.65.0 ] }
{ installations: [], versions: [], version: undefined }
tea: gnome.org/libxml2@2.10.3 locking
tea: gnome.org/libxml2@2.10.3 querying
{
  src: URL {
    href: "https://dist.tea.xyz/gnome.org/libxml2/darwin/aarch64/v2.10.3.tar.xz",
    origin: "https://dist.tea.xyz",
    protocol: "https:",
    username: "",
    password: "",
    host: "dist.tea.xyz",
    hostname: "dist.tea.xyz",
    port: "",
    pathname: "/gnome.org/libxml2/darwin/aarch64/v2.10.3.tar.xz",
    hash: "",
    search: ""
  },
  dst: ~/.tea/tea.xyz/var/www/gnome.org∕libxml2-2.10.3+darwin+aarch64.tar.xz
}
tea: gnome.org/libxml2@2.10.3 downloading
tea: gnome.org/libxml2@2.10.3 downloading 763 KiB
error: http (spilt-tea-404)
No such file or directory (os error 2), open '/Users/hasegawa/.tea/tea.xyz/var/www/gnome.org∕libxml2-2.10.3+darwin+aarch64.tar.xz'
{
  "0": {
    src: URL {
      href: "https://dist.tea.xyz/gnome.org/libxml2/darwin/aarch64/v2.10.3.tar.xz",
      origin: "https://dist.tea.xyz",
      protocol: "https:",
      username: "",
      password: "",
      host: "dist.tea.xyz",
      hostname: "dist.tea.xyz",
      port: "",
      pathname: "/gnome.org/libxml2/darwin/aarch64/v2.10.3.tar.xz",
      hash: "",
      search: ""
    },
    dst: ~/.tea/tea.xyz/var/www/gnome.org∕libxml2-2.10.3+darwin+aarch64.tar.xz,
    logger: Logger {
      prefix: "\x1b[38;5;238mtea:\x1b[39m \x1b[38;5;244mgnome.org/libxml2@2.10.3\x1b[39m ",
      lines: 1,
      last_line: "\x1b[38;5;86mdownloading\x1b[39m \x1b[38;5;244m763 KiB\x1b[39m",
      tty: [Function: tty] { text: [Function], getCursorPosition: [Function] },
      verbosity: 3,
      prefix_length: 29
    }
  },
  underr: NotFound: No such file or directory (os error 2), open '/Users/hasegawa/.tea/tea.xyz/var/www/gnome.org∕libxml2-2.10.3+darwin+aarch64.tar.xz'
    at async Object.open (deno:runtime/js/40_files.js:55:17)
    at async internal (file:///opt/tea.xyz/src/v0.13.5/src/hooks/useDownload.ts:56:27)
    at async download_with_sha (file:///opt/tea.xyz/src/v0.13.5/src/hooks/useDownload.ts:109:18)
    at async install (file:///opt/tea.xyz/src/v0.13.5/src/prefab/install.ts:41:42)
    at async install (file:///opt/tea.xyz/src/v0.13.5/src/app.exec.ts:352:25)
    at async default (file:///opt/tea.xyz/src/v0.13.5/src/app.exec.ts:20:38)
    at async file:///opt/tea.xyz/src/v0.13.5/src/app.ts:31:9
}
```

**Comment by AkiraHasegawa1997** [comment]
It seems to be solved by creating `www` directory before the execution as following:
```terminal
➜ tea +rust-lang.org      
tea: gnome.org/libxml2@2.10.3 downloading 763 KiB
error: http (spilt-tea-404)
No such file or directory (os error 2), open '/Users/hasegawa/.tea/tea.xyz/var/www/gnome.org∕libxml2-2.10.3+darwin+aarch64.tar.xz'

➜ mkdir ~/.tea/tea.xyz/var/www

➜ tea +rust-lang.org          
tea: installed: ~/.tea/gnome.org/libxml2/v2.10.3
tea: installed: ~/.tea/openssl.org/v1.1.118
tea: installed: ~/.tea/gnu.org/gettext/v0.21.1
tea: installed: ~/.tea/curl.se/v7.86.0
tea: installed: ~/.tea/perl.org/v5.36.0
tea: installed: ~/.tea/tea.xyz/gx/cc/v0.1.1
tea: installed: ~/.tea/zlib.net/v1.2.13
tea: installed: ~/.tea/git-scm.org/v2.38.1
tea: installed: ~/.tea/rust-lang.org/v1.65.0
tea: installed: ~/.tea/rust-lang.org/cargo/v0.65.0
this is a temporary shell containing the following packages:
gnome.org/libxml2@2.10.3, openssl.org@1.1.118, gnu.org/gettext@0.21.1, curl.se@7.86.0, perl.org@5.36.0, tea.xyz/gx/cc@0.1.1, zlib.net@1.2.13, git-scm.org@2.38.1, rust-lang.org@1.65.0, rust-lang.org/cargo@0.65.0
when done type: `exit'
tea ~ exit
```

**Comment by fabiotc** [comment]
Like @AkiraHasegawa1997 described here, once I manually created `~/.tea/tea.xyz/var/www`, everything works.

**Comment by skabber** [comment]
I had the same problem. The same, manual create '$HOME/.tea/tea.xyz/www' fixed it for me.

'Linux steamdeck 5.13.0-valve31-1-neptune #1 SMP PREEMPT Tue, 08 Nov 2022 22:43:07 +0000 x86_64 GNU/Linux'

**Comment by mxcl** [comment]
yeah this was a regression, 0.13.6 coming up. Sorry :/ we haven't enough tests and I unwisely merged #85

**Comment by mxcl** [comment]
k update pushed, should resolve this. Sorry for this embarrassing issue :(

`sh <(curl tea.xyz)` to update (or `tea --sync +tea.xyz`)
